Cargo.toml

```diff
@@ -15,6 +16,7 @@ hound = { version = "3.3.1", optional = true }
 lazy_static = "1.0.0"
 lewton = { version = "0.10", optional = true }
 minimp3 = { version = "0.3.2", optional = true }
+reqwest = { version = "0.9.0", default-features = false, features = ["rustls-tls"], optional = true }
```

Comment:
Could you remove the rustls-tls feature from here, copy the reqwest dependency and add it below as a dev-dependency, with rustls-tls enabled? That way, users of rodio have free choice over the tls implementation.